packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -164,10 +163,8 @@ const Tooltip: React.FC<TooltipProps> = (props: ScopedProps<TooltipProps>) => {
         // we dispatch here so `TooltipProvider` isn't required to
         // ensure other tooltips are aware of this one opening.
         document.dispatchEvent(new CustomEvent(TOOLTIP_OPEN));
-      } else {
-        isFocusOpenRef.current = false;
+        onOpen();
       }
-      window.clearTimeout(openTimerRef.current);
```

Comment:
This was only clearing if the `open` value changed from `false` to `true` but not if the trigger was focused and then the mouse entered it because when it was focused, open will have been true and mousing over it would have tried to set open to true again which will have ran the open timer.

Reply:
Good catch!